### PR TITLE
Add `scriptencoding` command

### PR DESCRIPTION
**Pull request by Fymyte**
Fixes: #91

### CODE REVIEW

File: grammar.js [hunk 1]
```diff
@@ -192,6 +194,71 @@ module.exports = grammar({
     normal_statement: ($) => bang_range_command($, 'normal', alias(/ .*/, $.commands)),
     startinsert_statement: ($) => maybe_bang($, tokalias($, 'startinsert')),
     stopinsert_statement: ($) => tokalias($, 'stopinsert'),
+
+    fileencoding: ($) =>
```
Comment (vigoux) [inline grammar.js:198]: Maybe name that `encoding` rather?
I am pretty sure we will be able to use that elsewhere...

Comment (Fymyte) [reply grammar.js:198]: I named it 'fileencoding' just because it is the name of the vim option, but I can change it if you prefer

Comment (vigoux) [reply grammar.js:198]: Yeah let's have it as `encoding` rather.

Comment (clason) [reply grammar.js:198]: There's also `encoding` (as well as  `printencoding`, `makeencoding`, `termencoding`) as well as `scriptencoding` (which is a command though).

Comment (vigoux) [reply grammar.js]: Yeah but we can't invent a word to describe the encoding, so...

**Pull request merged**